mobile/asa-go/src/components/map/asaGoMap.test.tsx

```diff
@@ -24,12 +24,14 @@ describe("ASAGoMap", () => {
   });
 
   const defaultProps = {
-    selectedFireCenter: undefined,
+    testId: "asa-go-map",
     selectedFireShape: undefined,
+    setSelectedFireShape: vi.fn(),
```

Comment:
Can we add tests that assert that this is called with the tapped fire shape? Can we also include assertions that test that `pop.setPosition` is called with the associated fire shape location?

Reply:
@conbrad  Any thoughts on how to do this? Working with/mocking openlayers map, layers and overlays is messing with my head!

Reply:
Thanks for pointing that out @brettedw. I could probably refactor the popup to follow a similar pattern and have props of ol.map and position which would get me part way there. In order to simulate a click on a feature in a vector tile layer I feel like I would also need to refactor the `AsaGoMap` to accept a vector tile layer prop (the fire zone unit layer) as a prop and then mock that. This would likely also allow us to assert on the advisory/warning state of the features in the layer.